book/src/intro.md

```diff
@@ -0,0 +1,39 @@
+# Introduction
+
+Proptest is a property testing framework (i.e., the QuickCheck family)
+inspired by the [Hypothesis](http://hypothesis.works/) framework for
+Python. It allows to test that certain properties of your code hold for
+arbitrary inputs, and if a failure is found, automatically finds the
+minimal test case to reproduce the problem. Unlike QuickCheck, generation
+and shrinking is defined on a per-value basis instead of per-type, which
+makes it more flexible and simplifies composition.
+
+If you have dependencies which provide QuickCheck `Arbitrary`
+implementations, see also the related
+[`proptest-quickcheck-interop`](https://crates.io/crates/proptest-quickcheck-interop)
+crates which enables reusing those implementations with proptest.
+
+## Status of this crate
+
+The majority of the functionality offered by proptest is in active use and
+is known to work well.
+
+The API is unlikely to see drastic breaking changes, but there may still be
+minor breaking changes here and there, particularly when "impl Trait"
+becomes stable and after the upcoming redesign of the `rand` crate.
```

Comment:
This bit has mostly happened?

I think maybe we should point to GATs instead now...